plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergParquetSystemTables.java

```diff
@@ -22,4 +29,31 @@ public TestIcebergParquetSystemTables()
     {
         super(PARQUET);
     }
+
+    @Test
+    public void testPropertiesTable()
+    {
+        try (TestTable table = newTrinoTable("test_properties_table", "AS SELECT 1 x")) {
+            assertThat(query("SELECT * FROM \"%s$properties\"".formatted(table.getName())))
+                    .matches("""
+                            VALUES (VARCHAR 'write.format.default', VARCHAR 'PARQUET'),
+                            (VARCHAR 'commit.retry.num-retries', VARCHAR '4'),
```

Comment:
There is no need to verify unrelated properties here. We could add a helper method like:
```java
    private Map<String, String> getTableProperties(String tableName)
    {
        return computeActual("SELECT key, value FROM \"" + tableName + "$properties\"").getMaterializedRows().stream()
                .collect(toImmutableMap(row -> (String) row.getField(0), row -> (String) row.getField(1)));
    }
```

Reply:
Fine idea, applied!